ReadMe.md

```diff
@@ -2,32 +2,54 @@
 ## Version
 > 1.3.9
 
+| Branch         | Status|  
+|:---------------|:---------------------------------------------------------------------------------------------------------------------------|  
+| `production`   | [![Build Status](https://travis-ci.org/NYPL/dgx-homepage.svg?branch=master)](https://travis-ci.org/NYPL/dgx-homepage)      |  
+| `qa`           | [![Build Status](https://travis-ci.org/NYPL/dgx-homepage.svg?branch=development)](https://travis-ci.org/NYPL/dgx-homepage) |  
```

Comment:
is this should be `[![Build Status](https://travis-ci.org/NYPL/dgx-homepage.svg?branch=qa)]`

and the next one should be 
[![Build Status](https://travis-ci.org/NYPL/dgx-homepage.svg?branch=development)]